internal/plugin/xns.go

```diff
@@ -583,8 +583,10 @@ func (svc *Manifest) genXNSActivitiesWorkflowMethod(f *g.File, workflow protoref
 						g.Qual(activityPkg, "RecordHeartbeat").Call(g.Id("ctx"), g.Id("run").Dot("ID").Call()),
 					),
 					g.Case(g.Op("<-").Id("ctx").Dot("Done").Call()).Block(
+						g.List(g.Id("disconnectedCtx"), g.Id("cancel")).Op(":=").Qual("context", "WithTimeout").Call(g.Qual("context", "Background").Call(), g.Qual("time", "Second").Op("*").Lit(5)),
```

Comment:
Why a 5 second timeout in particular? Does it make sense for this to be configurable?